### PR TITLE
edit readme.md file with our architecture

### PR DESCRIPTION
**Pull request by rotem-benzvi**

### CODE REVIEW

**Review by lioramilbaum — APPROVED** [review]
LGTM even though the PR description is empty.

**Comment by rotem-benzvi** [comment]
ok thanks! 
just to be sure, if my PR approved i do the merge pull request, or to wait that you will?

**Comment by lioramilbaum** [comment]
You can merge it

File: README.md [hunk 1]
```diff
@@ -1,7 +1,119 @@
-# givit
-## team members:
+<h1 align="center">
+  <img src="../givit/static/img/leaveit2givit.jpg" />
```
Comment (tbrisker) [inline README.md:2]: Looks like this didn't work - try to check in the [Guide](https://guides.github.com/features/mastering-markdown/) how to add an image to markdown.

**Pull request merged**